### ISSUE TITLE
macOS Volume Mounts use unsupported lsexattr operation

### ISSUE THREAD

**Issue by ananthb** [fyne-io/fyne-cross#153]
### Describe the bug:

fyne-cross on macOS tries to mount a volume with the selinux label :z which fails when using the podman engine.
The :z or :Z label must not be used with podman on macOS. 

https://github.com/containers/podman-compose/issues/509

### To Reproduce:

Steps to reproduce the behaviour:
1. Install `podman`
2. Run the command `fyne-cross linux --debug --no-cache --env GOPRIVATE="github.com/xxx/*" src/mycmd`
3. See error

### Example code:

I'm using one of the examples straight from this repo unmodified.

```go
	a := app.New()
	w := a.NewWindow("SysTray")

	if desk, ok := a.(desktop.App); ok {
		m := fyne.NewMenu("MyApp",
			fyne.NewMenuItem("Show", func() {
				w.Show()
			}))
		desk.SetSystemTrayMenu(m)
	}

	w.SetContent(widget.NewLabel("Fyne System Tray"))
	w.SetCloseIntercept(func() {
		w.Hide()
	})
	w.ShowAndRun()
```

### Device and debug info (please complete the following information):

<summary>Device info</summary>

 - **OS:** macOS
 - **Version:** Ventura
 - **Go version:** 1.19.4
 - **fyne-cross version:** 1.3.0
 - **Fyne version:** 2.2.4

<details><summary>Debug info</summary><br><pre>

```bash
fyne-cross linux --debug --no-cache src/mvcmd
[i] Target: linux/arm64
command.Context{Volume:volume.Volume{binDirHost:"/Users/ananth.b/src/xxx/fyne-cross/bin", cacheDirHost:"/Users/ananth.b/Library/Caches/fyne-cross", distDirHost:"/Users/ananth.b/src/xxx/fyne-cross/dist", tmpDirHost:"/Users/ananth.b/src/xxx/fyne-cross/tmp", workDirHost:"/Users/ananth.b/src/xxx"}, Architecture:"arm64", Engine:command.Engine{Name:"podman", Binary:"/opt/homebrew/bin/podman"}, Env:map[string]string{"CC":"aarch64-linux-gnu-gcc", "GOARCH":"arm64", "GOOS":"linux", "GOPRIVATE":"github.com/xxx/*"}, ID:"linux-arm64", LdFlags:[]string(nil), OS:"linux", Tags:[]string{"gles"}, AppBuild:"1", AppID:"", AppVersion:"1.0", CacheEnabled:false, DockerImage:"docker.io/fyneio/fyne-cross:1.3-linux-arm64", Icon:"/Users/ananth.b/src/xxx/Icon.png", Name:"wireapp", Package:"src/xxx", Release:false, StripDebug:true, Debug:true, Pull:false, Category:"", Certificate:"", Developer:"", Keystore:"", KeystorePass:"", KeyPass:"", Password:"", Profile:""}
[i] Cleaning target directories...
[✓] "bin" dir cleaned: /Users/ananth.b/src/xxx/fyne-cross/bin/linux-arm64
[✓] "dist" dir cleaned: /Users/ananth.b/src/xxx/fyne-cross/dist/linux-arm64
[✓] "temp" dir cleaned: /Users/ananth.b/src/xxx/fyne-cross/tmp/linux-arm64
[i] Checking for go.mod: /Users/ananth.b/src/xxx/go.mod
[✓] go.mod found
[i] Building binary...
/opt/homebrew/bin/podman run --rm -t -w /app -v /Users/ananth.b/src/xxx:/app:z --userns keep-id -e use_podman=1 -e CGO_ENABLED=1 -e GOCACHE=/go/go-build -e GOPRIVATE=github.com/xxx/* -e GOOS=linux -e GOARCH=arm64 -e CC=aarch64-linux-gnu-gcc -e CGO_LDFLAGS=-w -s docker.io/fyneio/fyne-cross:1.3-linux-arm64 go build -trimpath -tags gles -o /app/fyne-cross/bin/linux-arm64/xxx -v src/xxx
WARNING: image platform ({amd64 linux  [] }) does not match the expected platform ({arm64 linux  [] })
Error: preparing container 673a8c85431e41b5c68cbf9d1f656b791a5ac8110a699495a723f41376d76060 for attach: lsetxattr /Users/ananth.b/src/xxx/.dockerignore: operation not supported
[✗] exit status 126
```
</pre></details>

**Comment by Bluebugs** [comment]
This is addressed in PR #144 .

**Comment by Bluebugs** [comment]
This is fixed in develop.
